playbook.md

```diff
@@ -971,189 +971,72 @@ to build an appropriate hosting platform for their service. This is fair
+
 You can read more about GovPress at
 [www.govpress.com](https://www.govpress.com).
 
-We're in the process of building a new container-based hosting platform which
-will be able to support services that aren't built in WordPress. Watch this
-space.
+### Tailored support
+
+We also offer support for services that aren't built in WordPress. We are able to host and support services with our [tailored support plan for cloud applications](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/393702576050692), even if your service isn't hosted with us.
 
 ### Support helpdesk
 
 The dxw Technical Operations team build, manage and run GovPress, as well as
-provide first-line support.
+providing first-line support for our clients on tailored support plans.
 
 We use [Zendesk](https://dxw.zendesk.com) to manage support requests. Any
 incidents, requests for us to fix a problem or make a change to a site must
 come to us via Zendesk as a ticket. We use tickets to manage requests in order
 to:
 
-* Keep track of all the things we need to do, and what state they're in
+* Ensure oversight of all the things we need to do, and what state they're in, for us and our clients
 * Have a record of the changes we're asked to make
 * Ensure that we only accept change requests from people who are authorised
 * Generate data about how much staff time is spent on each client's issues.
 
 If a client asks us to do something in person, on the phone or via email, we
-politely ask them to visit the [dxw Zendesk](https://dxw.zendesk.com/) instead,
+ask them to visit the [dxw Zendesk](https://dxw.zendesk.com/),
 and submit a ticket there. This is so that:
 
 * We can keep track of the issue from report to solution, and ensure that it's
   assigned to the right person;
 * We don't risk misunderstanding the problem by writing it down based on a
   verbal explanation
 * We can formally track the changes that people ask for, by documenting them in
-  tickets that we can look at later if need to
-* We can be sure that requests are coming from people who are authorised to
-  submit them
+  tickets that we can look at later if we need to
 
 For these reasons, we do not do any work on a client website or service unless
-we are working on a development sprint or on a relevant ticket. We think this
+we are working during scheduled development time or on a relevant ticket. We think this
 is really important.
 
-#### Support sprints
-
-We work on tickets by having a developer on a support sprint. Developers are
-assigned to support for two weeks at a time and work on tickets for the
-duration of the sprint. If there is a ticket they need another developer to
-help with, this work is scheduled.
-
-#### Client experience (AKA: ticket principles)
```

Comment:
I like that we have these but agree that they need a bit of an update. Do we want to just trim them down for now and review with the team to sort out the detail, then we can update them properly later? I think the 'principles of how we do support' is a thing that should be in the playbook and I'm a little wary about removing it all now without having a commitment to replace them soon.

Reply:
I think we either add these to the Ops docs or keep them in the playbook. I'm hesitant to trim them down, without the support of the tech team, as these are principles that should be agreed by all. Perhaps we look to schedule a session within the next month to review these principles and update these in the playbook after that?

I've added the Client experience (AKA: ticket principles) back in, so these are unchanged for now.